tests/test_pipelines_common.py

```diff
@@ -436,16 +442,20 @@ def test_to_device(self):
         pipe = self.pipeline_class(**components)
         pipe.set_progress_bar_config(disable=None)
 
-        pipe.to("cpu")
+        pipe.to("cpu", dtype=torch.float32)
```

Comment:
nit: maybe change the name of this test too to something like `test_to_device_dtype` or `test_pipeline_to`?

Reply:
Sounds good, I'll update the test name!

I was going to include the to() dtype method in the checkpoint loading code, so if you had an original stable diffusion checkpoint saved with 32-bit weights, you could conveniently load it into 16-bit weight pipeline after the convert_from_ckpt code (I have a lot of these with the stable-diffusion-webui, and don't have VRAM to fully use 32-bit weights). 
There's probably not too many use cases but I thought it would be nice to save a loop through the pipeline components if you wanted to set the device and dtype together. The underlying components supported this call signature so it felt natural to extend it to the pipeline.